README.md

```diff
@@ -58,6 +58,16 @@ bundle exec jekyll build
 
 which will leave the results in a `_site` directory.
 
+### Running the link checker
+
+GitHub Actions checks each pull request for broken links using the
+[`htmlproofer`](https://github.com/gjtorikian/html-proofer)
+package.  If you wish to run `htmlproofer` locally, `cd` into the `submitty.github.io`
+repository and run the command:
+```
+htmlproofer ./_site --assume-extension --empty-alt-ignore --disable_external
```

Comment:
This should be prefixed with `bundle exec ...` as that's more portable than assuming than bundle always installs stuff into a bin folder.